Automattic/jetpack
# Backup: add connect screen to secondary admins

## Proposed changes:

* Add connect screen for secondary admins when the site has an active backup plan and has an owner connected.
  * After login, the secondary admin will be able to view the backup dashboard and storage details (if apply).

## Screenshots:
| Before | After |
|---|---|
| ![image](https://github.com/Automattic/jetpack/assets/1488641/9c70eacf-ce70-412c-ae9c-4e9db1007b36) | ![image](https://github.com/Automattic/jetpack/assets/1488641/7884ffa0-207d-48b7-9f5a-13690ffddb5c) |

## Known issues:
* The connect screen is not displaying the VaultPress Backup logo. It may require to modify [`ConnectScreen` component](https://github.com/Automattic/jetpack/blob/436e9244c303dc1bc7ac57bb258e4148998962e6/projects/js-packages/connection/components/connect-screen/basic/index.jsx#L26) to receive the logo, as [`ConnectScreenRequiredPlan`](https://github.com/Automattic/jetpack/blob/436e9244c303dc1bc7ac57bb258e4148998962e6/projects/js-packages/connection/components/connect-screen/required-plan/index.jsx#L14) does. It is better to address this in another PR given that we will be modifying another package.

### Other information:


## Jetpack product discussion


* peaFOp-FV-p2
* p6TEKc-6KA-p2

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Spin up a Jurassic Ninja site with just the Jetpack VaultPress Backup plugin.
* Connect the site to Jetpack using your main WordPress.com account and add a Jetpack VaultPress Backup plan.
* Ensure you are able to see the VaultPress Backup dashboard (main page).
* Add a new user to your blog.
* In a incognito window (or a new browser), log in with the new user.
* Navigate to VaultPress Backup page
* You should see a connect screen similar to the one shared above.
* Log in using a different WordPress.com account.
* After login, you should see the backup dashboard as the first user.
